README.md

```diff
@@ -1,2 +1,87 @@
-# HelloMicroverse
-This is hello Microverse project ( Module 1 )
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+🚀 Live Demo
+💻 Getting Started
+Setup
+Prerequisites
+Install
+Usage
+Run tests
+Deployment
+👥 Authors
+🔭 Future Features
+🤝 Contributing
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ (OPTIONAL)
+📝 License
+📖 Hello Microverse
+Describe your project in 1 or 2 sentences.
+
+Hello Microverse is a project that we made as a part of module 1
+
+🛠 Built With
+Tech Stack
+
+Client
+HTML / CSS
+Key Features
+This is Hello Microverse project part of module 1
+
+(back to top)
+
+🚀 Live Demo
+Soon
+
+Live Demo Link
+(back to top)
+
+💻 Getting Started
+Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+◦ Installation of Git
+◦ Installation of NPM
+◦ A working browser
+◦ Internet connection
+◦ A code editor like Visual Studio Code
```

Comment:
- [ ] Kindly highlight the header and sub-header for the readme section. :+1: 

Your Readme             |  Readme template
:-------------------------:|:-------------------------:
![readme](https://user-images.githubusercontent.com/70483835/218862691-24bfe753-07b5-4a14-ada5-e5db34ec06af.png) |  ![readme2](https://user-images.githubusercontent.com/70483835/218862723-86aac9ff-6186-4363-8f69-01c729be95aa.png)